be/src/util/dynamic_cache.h

```diff
@@ -303,10 +303,16 @@ class DynamicCache {
         return ret;
     }
 
+    void try_evict(size_t target_capacity) {
+        std::lock_guard<std::mutex> lg(_lock);
+        _evict(target_capacity);
+        return;
+    }
+
 private:
-    bool _evict() {
```

Comment:
Does `_evict()` trigger flush memory to L0 index file?

Reply:
> Will `_evict()` trigger flush memory to L0 index file?

No, evict just try remove entry in cache but not trigger flush L0. If we trigger flush in evict, we may cause large write amplification.